templatestore/frontend/src/components/Tinyurlcomp.js

```diff
@@ -0,0 +1,199 @@
+import React, { useState, useEffect } from 'react';
+import styles from '../style/templateScreen.less';
+import axios from 'axios';
+import { backendSettings } from './../utils.js';
+import Dropdowncomp from './Dropdowncomp';
+export default function Tinyurlcomp(props) {
```

Comment:
Seems like spacing and indentation are messed up here. Can you at least try improving the formatting with the help of https://google.github.io/styleguide/pyguide.html#3-python-style-rules?